### ISSUE TITLE
Update runtime to 23.08

### ISSUE THREAD

**Issue by AsciiWolf** [flathub/com.sublimetext.three#38]
Please, if this Flatpak is still maintained, update its runtime to 23.08. The currently used 21.08 is outdated and not supported anymore. Thanks!

/cc @ptomato

**Comment by ptomato** [comment]
Duplicate of #36, which is blocked on #37. Someone will have to figure that out first, or the upgrade is likely to break packages for all existing users.
